### ISSUE TITLE
Split query file and parallel on a cluster.

### ISSUE THREAD

**Issue by xiekunwhy** [mummer4/mummer#133]
Hi,

Can I split query file (one scaffold per file) and then map them to reference pallalelize (via qsub, for example) on a cluster? Can I get same results if I do so?

Best,
Kun

**Comment by alekseyzimin** [comment]
Yes, you can with the latest version and the results will be the same.
